src/main/java/org/hillel/homework/entity/OrderItem.java

```diff
@@ -0,0 +1,33 @@
+package org.hillel.homework.entity;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+import jakarta.persistence.*;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+@Entity
+@Table(name = "order_item")
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class OrderItem {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "product_id")
+    private Product product;
+
+    private BigDecimal quantity;
+    @ManyToOne
+    @JoinColumn(name = "order_id")
+    private Order order;
```

Comment:
do you really need order in order item?

Reply:
Yes, because this is a many-to-one relationship between the order item and the order.

Each order item belongs to an order, so it must contain this order (or, technically, the order ID).